baker/GDriveImagesBaker.tsx

```diff
@@ -25,7 +25,10 @@ export const bakeDriveImages = async (
     const images: Image[] = await db
         .knexRaw<DbRawImage>(
             knex,
-            `SELECT * FROM images WHERE id IN (SELECT DISTINCT imageId FROM posts_gdocs_x_images)`
+            `SELECT *
+             FROM images
+             WHERE id IN (SELECT DISTINCT imageId FROM posts_gdocs_x_images)
+                OR filename IN (SELECT DISTINCT relatedLinkThumbnail FROM posts_gdocs_tombstones)`
```

Comment:
Does this make sense, @ikesau?

Reply:
BTW, is subquery faster than a join, or why are we doing that?

Reply:
Will finish the review tomorrow but one thing that jumps out here is that we could get duplicates in `images` now, I think? Maybe `SELECT DISTINCT *` could help.

I don't know of a particular reason to subqueries over joins in this case. Feel free to change it to whichever's more legible.